index.js

```diff
@@ -188,5 +188,12 @@ const saveEditChanges = (event) => {
 
 };
 
+const valueCheck = (num1, num2) => {
+  if (num1 > num2) {
+    return "Num1 is Bigger then Num2";
+  }
+  return "Num1 is Smaller then Num2";
+}
```

Comment:
**Suggestion:** Consider using a more descriptive message in the return statements of the `valueCheck` function. Also, handle the case where `num1` and `num2` are equal. [enhancement]
```suggestion
const valueCheck = (num1, num2) => {
  if (num1 > num2) {
    return `The number ${num1} is greater than ${num2}`;
  } else if (num1 < num2) {
    return `The number ${num1} is smaller than ${num2}`;
  } else {
    return 'Both numbers are equal';
  }
}
```